theme/static/js/hs-vue/subject-keywords-app.js

```diff
@@ -20,7 +20,21 @@ let subjKeywordsApp = new Vue({
         document.getElementById('lst-tags').style.display = "block";
     },
     methods: {
+        onKeywordInput: function (event) {
+            let newKWLength =  this.newKeywordsLength();
+            if(newKWLength > 500){
+                this.newKeyword = this.newKeyword.slice(0, -1);
+                event.preventDefault();
+                this.error = "Warning: You have exceeded the character 500 character limit for the subject keywords field. In a future release you will be unable to continue adding keywords beyond the 500 character limit";
```

Comment:
you need to update this message to match the one that Clara suggested here:
https://github.com/hydroshare/hydroshare/issues/5638#issuecomment-2573443482

Reply:
The message should be:

> “the character limit for the subject keywords field is 500 characters, you have currently added (n). Please reduce keywords to below the character limit “